java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogSnapshotStack.java

```diff
@@ -136,7 +138,8 @@ private void createSnapshotCreationLambda(InstanceProperties instanceProperties,
         queue.grantSendMessages(snapshotCreationTrigger);
 
         snapshotCreationLambda.addEventSource(SqsEventSource.Builder.create(queue)
-                .batchSize(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_CREATION_BATCH_SIZE)).build());
+                .batchSize(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_CREATION_BATCH_SIZE))
+                .maxConcurrency(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_DELETION_LAMBDA_CONCURRENCY_MAXIMUM)).build());
```

Comment:
This should use the snapshot creation property, rather than the snapshot deletion property.

Reply:
Addressed